src/jobs/jobs.controller.ts

```
@@ -1,8 +1,10 @@
-import { Controller, Post, Body, Get, Param, Patch, UseGuards } from '@nestjs/common';
+import { Controller, Post, Body, Get, Param, Patch, Delete, UseGuards } from '@nestjs/common';
```

ESLint

[error] Replace `·Controller,·Post,·Body,·Get,·Param,·Patch,·Delete,·UseGuards·` with `⏎··Controller,⏎··Post,⏎··Body,⏎··Get,⏎··Param,⏎··Patch,⏎··Delete,⏎··UseGuards,⏎`

(prettier/prettier)